openresty-build-tools/kong-ngx-build

```diff
@@ -11,7 +11,7 @@ LUAROCKS_VER=
 PCRE_VER=
 FORCE=0
 OPENRESTY_PATCHES=$OPENRESTY_PATCHES
-OPENRESTY_PATCHES_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/openresty-patches
+OPENRESTY_PATCHES_DIR=$(realpath "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../openresty-patches)
```

Comment:
this script runs inside a docker container. Add a `COPY` to Dockerfile.openresty